src/main/cpp/hw/ctreadapters/pro/DragonVelocityRPSToCTREProAdapter.cpp

```diff
@@ -38,18 +38,11 @@ DragonVelocityRPSToCTREProAdapter::DragonVelocityRPSToCTREProAdapter(string netw
                                                                      int controllerSlot,
                                                                      const ControlData &controlInfo,
                                                                      const DistanceAngleCalcStruc &calcStruc,
-                                                                     DragonTalonFX &controller) : DragonControlToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
+                                                                     ctre::phoenixpro::hardware::TalonFX &controller) : DragonVelocityToCTREProAdapter(networkTableName, controllerSlot, controlInfo, calcStruc, controller)
 {
 }
 
 void DragonVelocityRPSToCTREProAdapter::Set(double value)
 {
     // TODO  Add phoenix pro commands
```

Comment:
Is this intentionally left as a no-op?

Reply:
Deleted override; base class handles this logic.  Thanks for the catch.